gax-httpjson/src/test/java/com/google/api/gax/httpjson/HttpJsonCallContextTest.java

```diff
@@ -152,4 +153,26 @@ public void testMergeWithTimeout() {
 
     Truth.assertThat(ctx1.merge(ctx2).getTimeout()).isEqualTo(timeout);
   }
+
+  @Test
+  public void testMergeWithTracer() {
+    ApiTracer explicitTracer = Mockito.mock(ApiTracer.class);
+    HttpJsonCallContext ctxWithExplicitTracer =
+        HttpJsonCallContext.createDefault().withTracer(explicitTracer);
+
+    HttpJsonCallContext ctxWithDefaultTracer = HttpJsonCallContext.createDefault();
+    ApiTracer defaultTracer = ctxWithDefaultTracer.getTracer();
+
+    // Explicit tracers override default tracers
```

Comment:
Does "default tracers" (as plural) make sense? I thought it is explicitly one (per context class type, but in this context I think it is not important).